tide/master/rest/RestInterface.cpp

```diff
@@ -176,6 +178,17 @@ void RestInterface::setupHtmlInterface( DisplayGroup& displayGroup,
     _impl->sceneController.reset( new RestController( _impl->httpServer.get(),
                                                       displayGroup ));
 
+    _impl->fileReceiver.reset( new FileReceiver( _impl->httpServer.get( )));
+
+    connect( _impl->fileReceiver.get(), &FileReceiver::open,
+             [this]( QString uri )
```

Comment:
This function is duplicated from lines 124 - 127 !! It should have been a class function instead of two lambdas IF there was a need to check for relative paths, but that's not the case since FileReceiver always emits open with an absolute path, so here it is enough to just forward the signal...